### PR TITLE
app-emulation/actool: create ebuild for actool and include in image

### PR DESCRIPTION
**Pull request by mischief**

### CODE REVIEW

**Comment by mischief** [comment]
depends on coreos/manifest#67

File: app-emulation/actool/actool-9999.ebuild [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+HOMEPAGE="https://github.com/appc/spec"
+SRC_URI=""
+
+LICENSE="Apache-2.0"
+SLOT="0"
+IUSE=""
+
+DEPEND=">=dev-lang/go-1.3"
+
+# TODO: Use or adapt coreos-go.eclass so we have half a chance of
```
Comment (marineam) [inline app-emulation/actool/actool-9999.ebuild]: You inherit coreos-go but don't use it. How come?

Comment (mischief) [inline app-emulation/actool/actool-9999.ebuild]: it's my first day. let me read that eclass and try again :^)

File: coreos-base/coreos/coreos-0.0.1.ebuild [hunk 2]
```diff
@@ -88,6 +88,7 @@ RDEPEND="${RDEPEND}
 	"
 
 RDEPEND="${RDEPEND}
+    app-emulation/actool
```
Comment (marineam) [inline coreos-base/coreos/coreos-0.0.1.ebuild]: tab plz

File: app-emulation/actool/actool-9999.ebuild [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+# cross-compiling builds working
+src_compile() {
+		CGO_ENABLED=0 ./build || die
+}
+
+src_install() {
+        dobin "${S}/bin/actool"
```
Comment (marineam) [inline app-emulation/actool/actool-9999.ebuild]: tab

File: app-emulation/actool/actool-9999.ebuild [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+LICENSE="Apache-2.0"
+SLOT="0"
+IUSE=""
+
+DEPEND=">=dev-lang/go-1.3"
+
+# TODO: Use or adapt coreos-go.eclass so we have half a chance of
+# cross-compiling builds working
+src_compile() {
+		CGO_ENABLED=0 ./build || die
```
Comment (marineam) [inline app-emulation/actool/actool-9999.ebuild]: one tab, not two

File: app-emulation/actool/actool-9999.ebuild (hunk 1 above)
Comment (marineam) [inline app-emulation/actool/actool-9999.ebuild]: How many first days do you get? :-P

**Comment by mischief** [comment]
my stupid editor decided to put 2 tab characters when i pressed tab once. confusion ensued. :\

File: app-emulation/actool/actool-9999.ebuild [hunk 5]
```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2015 CoreOS, Inc.
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+CROS_WORKON_PROJECT="appc/spec"
+CROS_WORKON_LOCALNAME="spec"
```
Comment (marineam) [inline app-emulation/actool/actool-9999.ebuild]: set below, delete this extra one :)

**Comment by mischief** [comment]
@marineam using coreos-go now. PTAL

**Comment by marineam** [comment]
[LGTM](http://lgtm.in/i/yDTNUm7bq)

File: coreos-base/coreos/coreos-0.0.1.ebuild [hunk 6]
```diff
@@ -88,6 +88,7 @@ RDEPEND="${RDEPEND}
 	"
 
 RDEPEND="${RDEPEND}
+	app-emulation/actool
```
Comment (marineam) [inline coreos-base/coreos/coreos-0.0.1.ebuild]: Oops, forgot to bump the symlink revision for this ebuild.

**Pull request merged**